### DISCUSSION TITLE
Rendering is fine locally, but the test environment has an error

### DISCUSSION THREAD

**Discussion by yebuq** [remarkjs/remark#1172] [Q&A] [answered]
![企业微信截图_eca6182d-e3c3-4c31-9856-737f842a715a](https://github.com/remarkjs/remark/assets/20119243/f1f91533-14ab-4f6c-9538-2fe8c77f971e)

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @Jaza914! 👋 
Sorry you ran into a spot of trouble.

> Rendering is fine locally, but the test environment has an error

Based off this, most likely there is a configuration issue in your build system.
The exact problem is unclear, there isn't enough information here.

Some suggestions, to help others to be able to offer more concrete and tangible help, from the [support guide](https://github.com/remarkjs/.github/blob/main/support.md):

> Help us help you! Spend time framing questions and add links and resources. Spending the extra time up front can help save everyone time in the long run. Here are some tips:
>
> * [Talk to a duck](https://rubberduckdebugging.com/)!
> * Don’t fall for the [XY problem](https://meta.stackexchange.com/questions/66377/what-is-the-xy-problem/66378#66378)
> * Search to find out if a similar question has been asked
> * Try to define what you need help with:
>     * Is there something in particular you want to do?
>     * What problem are you encountering and what steps have you taken to try and fix it?
>     * Is there a concept you don’t understand?
> * Provide sample code, such as a [CodeSandbox](https://codesandbox.io/) or video, if possible
> * Screenshots can help, but if there’s important text such as code or error messages in them, please also provide those as text
> * The more time you put into asking your question, the better we can help you

**Reply by yebuq** [reply]
thx for reply! i fixex this problem by import min files.likes this
<img width="783" alt="image" src="https://github.com/remarkjs/remark/assets/20119243/65812ed9-0ab3-4318-8d21-8a2b636f6150">
